nixos/modules/services/x11/display-managers/default.nix

```diff
@@ -321,14 +321,15 @@ in
         execCmd = mkOption {
           type = types.str;
           example = literalExample ''
-            "''${pkgs.lightdm}/bin/lightdm"
+            "''${pkgs.slim}/bin/slim"
           '';
           description = "Command to start the display manager.";
         };
 
         environment = mkOption {
           type = types.attrsOf types.unspecified;
           default = {};
+          example = { SLIM_CFGFILE = "/etc/slim.conf"; };
```

Comment:
I do not think it is good idea to include it in the example, as it will conflict with the SLiM module when used.